mkl: Use per-operator headers

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #75576
* #75575
* #75574
* #75573
* #75572
* #75571
* __->__ #75570
* #75569

Differential Revision: [D40126703](https://our.internmc.facebook.com/intern/diff/D40126703)

peterbell10 committed:
mkl: Use per-operator headers

[ghstack-poisoned]

peterbell10 committed:
Update on "mkl: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "mkl: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "mkl: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "mkl: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "mkl: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "mkl: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase and remove unneeded _meta.h ops headers on "mkl: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "mkl: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "mkl: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "mkl: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "mkl: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "mkl: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase on "mkl: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "mkl: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "mkl: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "mkl: Use per-operator headers"




[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40126703).

peterbell10 committed:
Rebase and fix merge conflicts on "mkl: Use per-operator headers"


Differential Revision: [D40126703](https://our.internmc.facebook.com/intern/diff/D40126703)

[ghstack-poisoned]
